setup.go

```diff
@@ -57,6 +58,16 @@ func setup(c *caddy.Controller) error {
 	return nil
 }
 
+func parseTTL(opt, arg string) (uint32, error) {
+	t, err := strconv.Atoi(arg)
+	if err != nil {
+		return uint32(t), err
+	}
+	if t < 0 || t > 3600 {
+		return uint32(t), fmt.Errorf("%s must be in range [0, 3600]: %d", opt, t)
```

Comment:
is 3600 a limit dictated by some rfc or something else?

Reply:
it is default came from plugin I based my work on

Reply:
So most DNS SOA's are 3600 MS Active Directory DNS and the infoblox one was also 3600, probably why coredns has it as well

Reply:
it is default came from plugin I based my work on